kobo/apps/help/models.py

```diff
@@ -43,6 +43,12 @@ class InAppMessage(AbstractMarkdownxModel):
     valid_from = models.DateTimeField(default=EPOCH_BEGINNING)
     valid_until = models.DateTimeField(default=EPOCH_BEGINNING)
     last_editor = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    project_ownership_transfer = models.ForeignKey(
```

Comment:
As per discussed, replaced the foreign key with a JSON field which would contain `object_name`: `object_id`.  It will help in the future if we need to display a specific in-app message for another app without creating another field in the model.